llvm/test/MC/RISCV/corev/simd-all-extensions.s

```diff
@@ -0,0 +1,2646 @@
+# RUN: llvm-mc -triple=riscv32 --mattr=+xcorevsimd -show-encoding %s 2>&1 \
```

Comment:
All tests that reference `xcorevsimd` will have to be changed to instead reference `xcvsimd`. I will not comment on any more instances of this.

Reply:
Fixed.

Reply:
Looks good.